doc/docs/modules/ROOT/pages/quickstart.adoc

```diff
@@ -177,6 +188,13 @@ every single node property as column prefixed by `source` or `target`
 |`sample`
 |No
 
+|`partitions`
+|This defines the parallelization level while pulling data from Neo4j.
+N.b. As More parallelization does not mean more performances so please tune wisely in according to
```

Comment:
"As more parallelization does not mean better performance, choose this setting wisely"

Note the interaction with "count".   I think what you mean is that if the count query returns 100 and partitions=5, you're going to be getting 5 partitions of 20 records, using skip/limit.   You should just say that.

Reply:
We should also mention that it only makes sense when you're pulling more than X (where X can be 1M or 10M)